TransferHelperApp/src/app/player/add-edit-player/add-edit-player.component.ts

```diff
@@ -7,9 +9,79 @@ import { Component, OnInit } from '@angular/core';
 })
 export class AddEditPlayerComponent implements OnInit {
 
-  constructor() { }
+  playerList$!:Observable<any[]>;
+  positionsList$!:Observable<any[]>;
+
+  constructor(private service:PlayerApiService) { }
+
+  @Input() player:any;
+  id: number = 0;
+  name: string = "";
+  surname: string = "";
+  birthdate: Date = new Date();
+  positionId!: number;
 
   ngOnInit(): void {
+    this.id = this.player.id;
+    this.name = this.player.name;
+    this.surname = this.player.surname;
+    this.birthdate = this.player.birthdate;
+    this.positionId = this.player.positionId;
```

Comment:
Nie wiem czy w przypadku w którym nie będzie żadnego gracza nie zwróci to błędu - wrzuciłbym rzeczy korzystające z `this.player` do if'a

Reply:
Dodano `if(this.player)`